app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -95,12 +99,12 @@ class BookmarksTest {
         }
     }
 
-    @Ignore("Intermittent failure on Nexus 6: https://github.com/mozilla-mobile/fenix/issues/8772")
     @Test
     fun createBookmarkFolderTest() {
         homeScreen {
         }.openThreeDotMenu {
         }.openBookmarks {
+            mDevice.waitForIdle(waitingTimeShort)
```

Comment:
Is this needed? Short time?

Reply:
I've seen this and the next test fail because of the "add folder" button not being visible, so I need something more here than the waitNotNull from clickAddFolderButton(). I also didn't want to make the test take too long so I tried the shorter wait to see how it goes. What would you suggest?

Reply:
I dropped the waitForIdle and changed the waitNotNull to wait for the content-description matcher (as Isabel recommended before). Please review it again.